sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2SQLSuite.scala

```diff
@@ -1909,7 +2020,6 @@ class DataSourceV2SQLSuite extends QueryTest with SharedSparkSession with Before
   }
 }
 
-
```

Comment:
Nit: please remove this unnecessary change as well

Reply:
Fixed. Thanks.